bin/strata-cli/src/seed/password.rs

```diff
@@ -77,6 +77,35 @@ impl Password {
             }
         }
     }
+
+    /// Validates the password strength and returns feedback if it's weak.
+    pub fn validate(&self) -> Result<(), String> {
+        let entropy = self.entropy();
+        if entropy.score() <= zxcvbn::Score::Two {
+            let feedback = entropy.feedback();
+            let feedback_message = if let Some(feedback) = feedback {
+                feedback
+                    .warning()
+                    .map(|w| w.to_string())
+                    .unwrap_or_default()
+            } else {
+                "".to_string() // empty string is fine
```

Comment:
An even cleaner approach could be to make `feedback_message` an `Option<String>`, but I wouldn't block on that.